conversations/push-ios/registernotifications/registernotifications.m

```diff
@@ -0,0 +1,17 @@
+if (self.updatedPushToken) {
+    [self.conversationsClient registerWithNotificationToken:self.updatedPushToken
+                                         completion:^(TCHResult *result) {
+      if (![result isSuccessful]) {
```

Comment:
This file has some lines indented by 4 spaces and some by 2 - is that intentional?

Reply:
No, I think this was the only file like that. Should be fixed now.